OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
My main reason for nerdishbynature/RequestKit#11 is to support setting labels (array of strings) and milestones (int) in issue creation and editing requests. GitHub’s API isn’t expecting strings for those fields.